speedcord/dispatcher.py

```diff
@@ -7,6 +7,7 @@
 
 
 class OpcodeDispatcher:
+    """Opcode dispatcher for discord API"""
```

Comment:
All the docstrings have not provided any value. Please follow examples such as [this](https://github.com/TAG-Epic/speedcord/blob/9d1b07ca5b6400930c796adba209b0a9ec48d6be/speedcord/client.py#L18)

Reply:
Okay I'll also add the params

Reply:
And please make the docstrings provide something that isn't in the class name or in the .pyi typings